ethstats/ethstats.go

```diff
@@ -316,11 +316,11 @@ func (s *Service) loop(chainHeadCh chan core.ChainHeadEvent, txEventCh chan core
 			}
 			// Keep sending status updates until the connection breaks
 			fullReport := time.NewTicker(15 * time.Second)
+			defer fullReport.Stop()
```

Comment:
I would say that this change is subtly wrong, or at least slightly suboptimal. Basically, you have here
```
for{ 
   fullReport := ...
   defer fullReport.Stop()
   /// do things
}
```
If this loops 1000 times, you have piled 1000 closures of deferred functions onto the callstack, but none of them have yet been actually `Stop`:ped. The way it is written means that each loop iteration will clean up it's own ticker, before the next one is created